devimint/src/gatewayd.rs

```diff
@@ -121,7 +121,7 @@ impl Gatewayd {
     pub async fn get_info(&self) -> Result<serde_json::Value> {
         retry(
             "Getting {} gateway info via gateway-cli info",
-            fedimint_core::util::FibonacciBackoff::default()
+            backon::FibonacciBuilder::default()
```

Comment:
Any reason we have a default / any reason we don't use it here?

Reply:
The `Default` trait appears to be the only way to construct any of the builders in the `backon` crate.

Reply:
Oh, so we only re-export the whole thing. Nvmd.

Reply:
I did that in this PR. Currently in master we're exporting only the `backon` builder structs, but we're renaming them. In this PR I removed the renamed export and am simply exporting the entire crate.